resources/qml/StatusIndicator.qml

```diff
@@ -26,32 +27,30 @@ Rectangle {
         }
     }
 
-    MouseArea {
-        id: ma
-
-        anchors.fill: parent
-        hoverEnabled: true
+    onClicked: {
+        if (state == MtxEvent.Read)
+            TimelineManager.timeline.readReceiptsAction(eventId);
```

Comment:
Can't you use model.id here?

Reply:
Yes, and I also was able to remove the `state` property. Thanks for pointing this out!